controllers/usersController.js

```diff
@@ -47,6 +47,17 @@ const getUser = async (req, res) => {
   }
 }
 
+/**
+ * Fetches the data about logged in user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getSelfDetails = (req, res) => {
+  res.send(req.userData.user)
```

Comment:
- Add safe checks to check if the userData actually exists.

Reply:
return res.json(...)
Ideally the CI should have failed because of no return statement here. 🤔

Reply:
Add Tests.

Reply:
I have added  the safe checks and return statement. I will add the tests soon.